app/routers/posts_router.py

```diff
@@ -30,16 +38,27 @@ async def create_post(
     return posts
 
 
-@router.get("/posts/{post_id}", response_model=PostResponse)
+@router.get(
+    "/posts/{post_id}",
+    response_model=PostResponse,
+    responses={404: {"description": "Post not found"}},
+)
 async def get_post(post_id, db: Session = Depends(get_db)):
     posts = crud.get_post_by_id(db, post_id)
     if posts is None:
         raise HTTPException(status_code=404, detail="Post not found")
     return posts
 
 
-@router.patch("/posts/{post_id}", response_model=PostResponse)
-async def edite_post(
+@router.patch(
+    "/posts/{post_id}",
+    response_model=PostResponse,
+    responses={
+        404: {"description": "Post not found"},
+        401: {"description": "Not authenticated, you must have Admin permission"},
```

Comment:
Didn't notice that earlier, but I think users must be allowed to edit their own posts. Is that something in the scope of this PR, or are we moving this change to a later PR?

Reply:
This response is send to user if he want to edit someone else post.

Reply:
I see!